src/client/pfs.go

```diff
@@ -673,16 +673,20 @@ func (c APIClient) PutFileURL(repoName string, commitID string, path string, url
 
 // CopyFile copys a file from one pfs location to another. It can be used on
 // directories or regular files.
-func (c APIClient) CopyFile(srcRepo, srcCommit, srcPath, dstRepo, dstCommit, dstPath string, overwrite bool) error {
-	if _, err := c.PfsAPIClient.CopyFile(c.Ctx(),
-		&pfs.CopyFileRequest{
-			Src:       NewFile(srcRepo, srcCommit, srcPath),
-			Dst:       NewFile(dstRepo, dstCommit, dstPath),
-			Overwrite: overwrite,
-		}); err != nil {
-		return grpcutil.ScrubGRPC(err)
+func (c APIClient) CopyFile(srcRepo, srcCommit, srcPath, dstRepo, dstCommit, dstPath string, overwrite bool, tag ...string) (retErr error) {
+	defer func() {
+		retErr = grpcutil.ScrubGRPC(retErr)
+	}()
```

Comment:
This seems a little over-engineered, there's only one return site for the error.

Reply:
I think this is a good pattern to follow for maintainability reasons. It may be simple to just wrap the error at the return site in this case, but we may add code later which would need to account for this error transform. Even if we don't, I would prefer to be consistent with the other wrappers (the ones I have changed) and follow the same pattern.